static-assets/components/cstudio-dialogs/new-template.js

```diff
@@ -41,7 +41,7 @@ CStudioAuthoring.Dialogs.NewTemplate = CStudioAuthoring.Dialogs.NewTemplate || {
 	 */
 	createDialog: function(path) {
 		var CMgs = CStudioAuthoring.Messages;
-		var previewLangBundle = CMgs.getBundle("siteDropdown", CStudioAuthoringContext.lang);
+        var siteDropdownLangBundle = CMgs.getBundle("siteDropdown", CStudioAuthoringContext.lang);
```

Comment:
Why is the bundle variable called siteDropdownLangBundle.  This code has nothing to do with siteDropdown.  That's copy paste and it makes the code hard to read

Reply:
Maybe this label needs to moved to a more generic bundle.  Assuming the var name applies to the bundle. So the real issue is this resource is in the wrong bundle.